tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+		"target": "ESNext",
+		"strict": true,
+    "noImplicitAny": true,
```

Comment:
`noImplicitAny` defaults to `true` when `strict` is set:

https://www.typescriptlang.org/tsconfig#noImplicitAny

Reply:
Removed in e1afcf0627b2414999063c1e83317e3b08b8a85e